app/assets/javascripts/spree/frontend/affirm_checkout.js

```diff
@@ -0,0 +1,74 @@
+(function(){
+  /* only include this setup once */
+  if (!window.AffirmPaymentMethods) {
+
+    /*****************************************************\
+        set the shared checkout data
+    \*****************************************************/
```

Comment:
Can we provide a `SolidusAffirm` object in js instead? That way we can simply `new SolidusAffirm(stuff).checkout()`
etc.